src/selectors/generateChannelLink.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Returns a link depending on the channel
+ * Depending on pageUrl, will either append to existing query params or create a new query
+ * @param {string} pageURL - Page url.
+ * @param {string} integrationName - Will append channel depending on integration name.
+ */
+
+export function generateChannelLink(pageURL, integrationName) {
+  let channelName;
+  switch (integrationName) {
+    case 'Magento':
+      channelName = 'Integration: Magento 2';
+      break;
+    case 'cpanel':
+      channelName = 'Integration: CPanel';
+      break;
+    case 'wordpress':
+      channelName = 'Integration: WordPress';
+      break;
+    default:
+      channelName = '';
+  }
```

Comment:
John mentioned allowing to make this dynamic which makes a lot sense. If we the code stays like this for some reason, I'd suggest something like "Cloudflare Plugin Frontend". Talk to  Garrett for this